### DISCUSSION TITLE
Branch select dialog has no way to search branches.

### DISCUSSION THREAD

**Discussion by thepont** [community/community#129262] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I feel like there was an update that was just rolled out that removes the ability to search for branches when creating pull requests, as you can see from the screen shot below this is different from the repo branch select that allows users to search for a branch by name or manually enter a branch.

This makes it quite difficult to create pull requests if the repository you're pulling into has a lot of branches.

## Pull Request Branch Select

<img width="383" alt="Screenshot 2024-06-21 at 12 56 01" src="https://github.com/community/community/assets/2901992/03c92385-409c-49f0-bb91-7312c14f4ade">

## Much better repo branch select.

<img width="354" alt="Screenshot 2024-06-21 at 12 56 17" src="https://github.com/community/community/assets/2901992/8b1ebd7d-6df0-4316-ab6c-d6725e1a09a0">

**Comment by Ganthology** [comment]
Agreed, my team still need this feature as well. Please bring it back.

**Comment by ghost** [comment]
Same my issue

**Comment by digismack** [comment]
https://github.com/orgs/community/discussions/129256

**Comment by annguyen1892** [comment]
Why https://www.githubstatus.com -> Pull Requests  not update this issue? the reason because it's not a critical issue

**Reply by muhnashir** [reply]
Fridays are not the best for Pull Request😁

**Comment by jayfaaa** [comment]
alternative :

[repolink]/compare/[Base]...[compare]

example
https://github.com/my_company/my_repo/compare/my_base...my_compare

**Comment by ChrisPeiSC** [comment]
same

**Comment by YuBoxiang** [comment]
Brewing a pay-to-use feature, Hmmm...

**Reply by ChrisPeiSC** [reply]
agree

**Comment by neilferreira** [comment]
cc @thepixelninja

**Comment by zhaochuninhefei** [comment]
same

**Comment by tejasraj-jar** [comment]
Same

**Comment by ajitstylishop** [comment]
Same issue even i am facing.In my repo there are 500+ branches now its difficult to search my one  because earlier i was searching by doing copy paste my branch name .

**Comment by eunion** [comment]
same

**Comment by LaiTaiYuan** [comment]
same

**Comment by borisFcns** [comment]
please bring it back, scrolling to find the branch is tedious !

**Comment by mrzeszutek-clari** [comment]
The project I'm working in has some 3.5k branches, lots of them made by automated processes. Do you expect me to find the one I'm looking fore by scrolling through all of them? This is nonsense, why are you removing a perfectly fine feature?

**Comment by thepont** [comment]
This seems to have been resolved..

![image](https://github.com/community/community/assets/2901992/351e222a-dc15-46ec-b0f9-9e6145a24508)
